modules/tinymce/src/plugins/table/main/ts/actions/TableActions.ts

```diff
@@ -42,8 +42,8 @@ export interface TableActions {
   pasteRowsAfter: AdvancedPasteTableAction;
   setTableCellType: SimpleTableAction;
   setTableRowType: SimpleTableAction;
-  makeColumnHeader: ElementTableAction;
-  unmakeColumnHeader: ElementTableAction;
+  makeColumnHeader: CombinedTargetsTableAction;
```

Comment:
Do we have an automated test that covers this change? If not, then can you please add one?

Reply:
Which change do you mean exactly?

Reply:
I could have sworn I replied to this... anyways the change is that you've changed what data needs to be passed to Snooker now from within the table plugin. So we need to make sure we have a test that covers making a selection across multiple columns a header. I know you have the snooker tests, but they are unit test, not functional tests. 

- [ ] Make sure we have a test in the table plugin that covers the changes made